gallery/src/main.scss

```diff
@@ -9,41 +9,63 @@
 @import '~@sonatype/react-shared-components/scss-shared/nx-container-helpers';
 @import '~@sonatype/react-shared-components/scss-shared/nx-flexbox-helpers';
 
+.nx-page-sidebar.gallery-page-sidebar {
+  padding-right: $nx-spacing-md;
+  padding-left: $nx-spacing-md;
```

Comment:
Why are we deviating from the usual padding here?  Or should the usual padding be updated?

Reply:
> Why are we deviating from the usual padding here? Or should the usual padding be updated?

The usual padding should be updated. I was hesitating to change the `nx-sidebar` default padding because of downstream effects (IQ uses it in one place). But I think that was a mistake and I should have.